README.md

```diff
@@ -99,7 +108,8 @@ Imagine that you take the code from this question, save it to a file called `sus
 
 What would be the output? Explain your answer.
 
-<!-- Replace this comment with your answer -->
+"Sushi is tasty"
```

Comment:
Your answer is correct; however your comment is wrong. If you put in a negative value, you will reach line 100 (` else if (x >= 3) {`) and this will trigger an error when the program tries to evaluate `x`.